fb_apt/recipes/default.rb

```diff
@@ -0,0 +1,73 @@
+#
+# Cookbook Name:: fb_apt
+# Recipe:: default
+#
+# Copyright 2014, Davide Cavalca
+#
+
+unless node.debian? || node.ubuntu?
+  fail 'fb_apt is only supported on Debian and Ubuntu.'
+end
+
+package 'apt' do
+  action :upgrade
+end
+
+# This takes precedence over anything in /etc/apt/apt.conf.d. We can't just
+# clobber that as several packages will drop configs there.
+template '/etc/apt/apt.conf' do
+  source 'apt.conf.erb'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  notifies :run, 'execute[apt-get update]'
+end
+
+# No sane package should drop stuff here, and bad preferences can seriously
+# mess up a machine, so let's clobber it to be safe.
+Dir.glob('/etc/apt/preferences.d/*').each do |f|
```

Comment:
yeah I went back and forth on this, but I don't think there's any legit reason to leave stuff there unmanaged. I'll add a note to the README about it